packages/core/src/components/skeleton/_skeleton.scss

```diff
@@ -0,0 +1,67 @@
+// Copyright 2016 Palantir Technologies, Inc. All rights reserved.
+// Licensed under the BSD-3 License as modified (the “License”); you may obtain a copy
+// of the license at https://github.com/palantir/blueprint/blob/master/LICENSE
+// and https://github.com/palantir/blueprint/blob/master/PATENTS
+
+@import "../../common/variables";
+
+.pt-skeleton {
+  border-radius: 2px;
+  background-color: rgba($gray1, 0.2);
+  width: 100%;
+  height: $pt-grid-size * 0.6;
+
+  // negative animation delay ensures animations are synced
```

Comment:
to clarify for myself, your vote is to disable scrolling in tables when it's loading? do we also disable scrolling if a single column is loading?